config/puma/dev.rb

```diff
@@ -0,0 +1,2 @@
+workers 2
```

Comment:
Did not fix a related issue around running rails and `ember build -w` at the same time. Will likely remove.